messages/web.login.md

```diff
@@ -0,0 +1,31 @@
+# summary
+
+authorize an org using the web login flow
+
+# description
+
+authorize an org using the web login flow
+If you specify an --instance-url value, this value overrides the sfdcLoginUrl value in your sfdx-project.json file. To specify a My Domain URL, use the format MyDomainName.my.salesforce.com (not MyDomainName.lightning.force.com). To log in to a sandbox, set --instanceurl to https://MyDomainName--SandboxName.sandbox.my.salesforce.com.
+To open in a specific browser, use the --browser parameter. Supported browsers are "chrome", "edge", and "firefox". If you don't specify --browser, the org opens in your default browser.
+
+# examples
+
+- $ <%= config.bin %> <%= command.id %> -a TestOrg1
+
+- $ <%= config.bin %> <%= command.id %> -i <OAuth client id>
+
+- $ <%= config.bin %> <%= command.id %> -r https://MyDomainName--SandboxName.sandbox.my.salesforce.com
+
+- $ <%= config.bin %> <%= command.id %> -a TestOrg1 -b firefox
+
+# browser
+
+browser where the org opens
+
+# deviceWarning
+
+auth:web:login doesn't work when authorizing to a headless environment. Use auth:device:login instead.
```

Comment:
Should we start using spaces instead of colons in messages? I know both work, just wondering if that will start pushing users towards the new style (also new autocomplete will only work with spaces).